### PR TITLE
Add new classes from glib 2.72

### PR DESCRIPTION
**Pull request by jf2048**
- `GDebugController`
- `GDebugControllerDBus`
- `GBindingGroup`

`GSignalGroup` can't really get a safe binding until this: https://gitlab.gnome.org/GNOME/glib/-/merge_requests/2521

### CODE REVIEW

File: gio/src/auto/debug_controller_dbus.rs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                Err(from_glib_full(error))
+            }
+        }
+    }
+}
+
+pub trait DebugControllerDBusExt: 'static {
+    #[cfg(any(feature = "v2_72", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_72")))]
+    fn connection(&self) -> Option<DBusConnection>;
```
Comment (bilelmoussaoui) [inline gio/src/auto/debug_controller_dbus.rs:57]: Probably not nullable

File: glib/src/gobject/auto/binding_group.rs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+crate::wrapper! {
+    #[doc(alias = "GBindingGroup")]
+    pub struct BindingGroup(Object<gobject_ffi::GBindingGroup>);
+
+    match fn {
+        type_ => || gobject_ffi::g_binding_group_get_type(),
+    }
+}
+
+impl BindingGroup {
```
Comment (bilelmoussaoui) [inline glib/src/gobject/auto/binding_group.rs:26]: Should probably manually implement like the current GBinding?

Comment (jf2048) [reply glib/src/gobject/auto/binding_group.rs:26]: Yes, jut realized I forgot that. At the very least `bind` needs it

File: gio/src/auto/debug_controller_dbus.rs (hunk 1 above)
Comment (jf2048) [reply gio/src/auto/debug_controller_dbus.rs]: Implemented it manually

File: glib/src/gobject/auto/binding_group.rs (hunk 2 above)
Comment (jf2048) [reply glib/src/gobject/auto/binding_group.rs:24]: Copied most of the implementation from `BindingBuilder`

File: gio/src/auto/debug_controller_dbus.rs (hunk 1 above)
Comment (sdroege) [reply gio/src/auto/debug_controller_dbus.rs]: You can set `nullable = false` in `Gir.toml` instead

File: gio/Gir.toml [hunk 3]
```diff
@@ -553,6 +554,15 @@ status = "generate"
     name = "no_match_rule"
     version = "2.70"
 
+[[object]]
+name = "Gio.DebugControllerDBus"
+status = "generate"
+manual_traits = ["DebugControllerDBusExtManual"]
+    [[object.property]]
+    name = "connection"
```
Comment (bilelmoussaoui) [inline gio/Gir.toml:562]: hmm, not sure how it ended up marking the property as nullable :/ maybe it should be but then the constructor isn't so...

Comment (sdroege) [reply gio/Gir.toml:562]: I guess pointer properties are nullable by default unless overridden in the docs?

Comment (jf2048) [reply gio/Gir.toml:562]: Sadly using `(not nullable)` does not work on properties, it would be nice if it did

**Comment by jf2048** [comment]
This can wait until closer to the 2.72 release

**Review by sdroege — APPROVED** [review]
Looks good to me. Why wait for 2.72 release?

**Comment by jf2048** [comment]
This is broken for me as long as the sysdep version depends on 2.72, the pkg-config version is still only 2.71 until release. But I guess it doesn't matter much because CI is not using that anyway?

**Comment by sdroege** [comment]
> the pkg-config version is still only 2.71 until release

Hrm yes, it should only require 2.71.X right now and not a non-existing 2.72. Can you submit a PR?

**Comment by jf2048** [comment]
Yes, this is ready

**Pull request merged**